exercicios/para-casa/emilly-santos/entrega.js

```diff
@@ -0,0 +1,68 @@
+function Cat(name, age, color, castrated) {
+    this.name = name;
+    this.age = age;
+    this.color = color;
+    this.castrated = castrated
+    this.historic = new MedicalHistory();
+}
+
+Cat.prototype.meow = function() {
+    console.log(`O gato ${this.nome} está miando`)
+}
+
+function Dog(name, age, color, castrated, race) {
+    this.name = name;
+    this.agr = age;
+    this.color = color;
+    this.castrated = castrated;
+    this.race = race;
+    this.historic = new MedicalHistory();
+}
+
+Dog.prototype.bark = function() {
+    console.log(`O gato ${this.nome} está latindo`)
+}
+
+function ExoticAnimal(name, age, color, species, adult) {
+    this.name = name;
+    this.age = age;
+    this.color = color;
+    this.species = species;
+    this.adult = adult;
```

Comment:
O **this.adult** precisa receber a função **ehAdulto** para que aqui apareça o returno da função.
Fica assim: "this.adult = this.ehAdulto()".